ga4gh/cli.py

```diff
@@ -266,10 +269,10 @@ def server_main(parser=None):
             description="GA4GH reference server")
     addGlobalOptions(parser)
     args = parser.parse_args()
-    frontend.configure(args.config_file, args.config)
+    frontend.configure(args.config_file, args.config, args.oidc_config_file)
     frontend.app.run(
-        host="0.0.0.0", port=args.port,
-        use_reloader=not args.dont_use_reloader)
+        host="localhost", port=args.port,
+        use_reloader=not args.dont_use_reloader, ssl_context="adhoc")
```

Comment:
Do you want to use ssl when there's no oidc_config_file? It sounds sensible, but I'm just observing that it might break some existing urls for people doing local development & testing, and bother them with certificate warnings.